nixos/lib/utils.nix

```diff
@@ -4,6 +4,8 @@ rec {
 
   # Check whenever fileSystem is needed for boot
   fsNeededForBoot = fs: fs.neededForBoot
+                        # If you change the list below, please update the
+                        # documentation for fileSystems.<name?>.neededForBoot
                      || elem fs.mountPoint [ "/" "/nix" "/nix/store" "/var" "/var/log" "/var/lib" "/etc" ];
```

Comment:
how about add a variable that can be accessed at `utils.*` and use that in the option description?

Reply:
Good idea, that makes it easier to maintain the description. By "variable" I guess you just mean a constant, right? I guess you are not proposing to make this user-customizable (since the user can just use `fileSystems.<name?>.neededForBoot` if they want to add to this list).

Reply:
example
```
lib.concatStringsSep " " (pkgs.lib.forEach utils.somename (i: "<filename>${i}</filename>"))
```

Reply:
> Good idea, that makes it easier to maintain the description. By "variable" I guess you just mean a constant, right?

Yep, like a constant.

Reply:
Done.